src/Intacct/AbstractClient.php

```diff
@@ -86,7 +86,7 @@ public function __construct(ClientConfig $config = null)
      *
      * @return Xml\OnlineResponse
      */
-    protected function executeOnlineRequest(array $functions, RequestConfig $requestConfig = null)
+    protected function executeOnlineRequest(array $functions, RequestConfig &$requestConfig = null)
```

Comment:
why does it need to be a reference for `requestConfig`?

Reply:
it's being done as a reference so to change as little of the existing code as possible

Reply:
when executeOnlineRequest is being leveraged in our SDK, it's not passing in a requestConfig and then generates one within executeOnlineRequest, but for executeBatch, it'll have a requestConfig passed up to that for it's reference as well since executeBatch calls executeOnlineRequest